### ISSUE TITLE
PRC ill-definition of face_number in 8.8.9.6.1

### ISSUE THREAD

**Issue by mvrhel** [pdf-association/pdf-issues#429]
The compressed tessellation object includes compression of the face normals.  The specification states "face_number is derived from the maximum value in triangle_face_array"  but does not tell how it is derived, nor what the use of this value is.  I suspect that one looks through the triangle_face_array values and picks the max and adds one to that to get the number of faces.  Then this value is used for the number of elements in the is_face_planar array.  But this is just conjecture and needs clarification in the spec.   In a similar manner, the is_point_color_on_face array size, is_multiple_line_attribute_on_face array size, and face_has_texture array size are not well defined in the same object in, but I am guessing they also use the derived face_number.   Again, clarification on this and how the face_number is derived rather than just saying it is derived would be good.

**Comment by mvrhel** [comment]
Yes it is not part of the data structure but rather derived from it.  I think my comment that it is one plus the max index found in triangle_face_array is correct.   This is more precise than just saying it is derived from....

**Comment by petervwyatt** [comment]
Note to PL: After correcting the definition, consider changing some of the empty array of sizes (`[]`) to _face_number_
